.github/workflows/test.yml

```diff
@@ -0,0 +1,29 @@
+name: Test
+on: [push, pull_request]
+jobs:
+  test-format:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.18.x
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Format
+      run: diff -u <(echo -n) <(gofmt -s -d .)
```

Comment:
I propose we not require `-s`. Sometimes it insists on removing helpful types.

Reply:
Done.